.travis.yml

```diff
@@ -2,6 +2,5 @@ language: node_js
 node_js:
     - 6   # node v6, to match jenkins
 install:
+    - scripts/fetch-develop.deps.sh
```

Comment:
It empirically does, although only did it this way to avoid installing copies of thing we then blow away again.